IdentityCore/src/requests/broker/MSIDSSOExtensionInteractiveTokenRequest.m

```diff
@@ -104,9 +110,27 @@ - (void)executeRequestWithCompletion:(MSIDInteractiveRequestCompletionBlock)comp
          }
         
         NSError *localError;
+        NSDictionary *enrollmentIds = [self.enrollmentIdsCache enrollmentIdsJsonDictionaryWithContext:self.requestParameters
+                                                                                                error:&localError];
+        if (localError)
```

Comment:
Should we consider this error as non critical?

Reply:
done